db/seeds.rb

```diff
@@ -1,7 +1,18 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+5.times do
+  user = User.create!(
+  email: Faker::Internet.email,
+  password: Faker::Internet.password
+  )
+end
+users = User.all
+
+25.times do
+  Item.create!(
+  user: users.sample,
+  name: Faker::Hipster.sentence
```

Comment:
probably best to indent these two lines since they are arguments to `Item.create`